lilac2/lilacpy.py

```diff
@@ -2,16 +2,15 @@
 import contextlib
 import importlib.util
 from pathlib import Path
-from typing import Generator, cast, Dict, Tuple
+from typing import Generator, cast, Dict
 
-from .typing import LilacMod, LilacMods, ExcInfo
+from .typing import LilacMod, ExcInfo
 from .lilacyaml import load_lilac_yaml, ALIASES
 
-def load_all(repodir: Path) -> Tuple[LilacMods, Dict[str, ExcInfo]]:
-  mods = {}
+def load_all(repo) -> Dict[str, ExcInfo]:
```

Comment:
`repo` is not needed. Return `mods` is better, e.g. we don't need to construct a `Repo` when we only care about the mods.